test/test_view.c

```diff
@@ -29,6 +29,7 @@ DEALINGS IN THE SOFTWARE.  */
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
What was getopt.h required for? I think this is meant to be for the GNU getopt_long extension, which POSIX getopt coming from unistd.h.

Which OS were you compiling on and with which compiler?